lib/Capabilities.php

```diff
@@ -114,7 +114,10 @@ public function getCapabilities(): array {
 				],
 				'conversations' => [],
 				'previews' => [
-					'max-gif-size' => (int)$this->serverConfig->getAppValue('spreed', 'max-gif-size', '3145728')
+					'max-gif-size' => (int)$this->serverConfig->getAppValue('spreed', 'max-gif-size', '3145728'),
+				],
+				'signaling' => [
+					'session-ping-limit' => (int)$this->serverConfig->getAppValue('spreed', 'session-ping-limit', '0'),
```

Comment:
```suggestion
					'session-ping-limit' => (int)$this->serverConfig->getAppValue('spreed', 'session-ping-limit', '200'),
```

Should we default enable with 200? @fancycode

Reply:
Yeah, 200 is probably a good start. We can tweak this later once you have some performance numbers.